src/main/java/rest/CountryResource.java

```diff
@@ -85,14 +117,19 @@ public String getCountryNameByAlpha2(@PathParam(value = "alpha2") final String a
     @Path("/events")
     @Produces(MediaType.APPLICATION_JSON)
     @GET
+    @Operation(summary = "Get all events in a given city, on given date/dates.",
+            tags = {"Events"},
+            responses = {
+                @ApiResponse(
+                        content = @Content(mediaType = "application/json", schema = @Schema(implementation = EventDTO.class))),
+                @ApiResponse(responseCode = "200", description = "The Requested list of events"),
+                @ApiResponse(responseCode = "400", description = "TODO - [INSERT RIGHT ERROR MESSAGE]")})
```

Comment:
What should this be? 

The description.

Reply:
The reason why this error was thrown.